### PR TITLE
Add backup container for recurring dropbox backup

### LINKED ISSUE #1: Backup Script
Add a periodic backup script that makes both local and online backups just like the one on the [wiki](https://github.com/metakgp/metakgp-wiki/blob/master/jobs/backup/backup_to_dropbox.py).

### PR DESCRIPTION
**Pull request by shikharish**
Closes #1

### CODE REVIEW

File: .env.template
Comment (harshkhandeparkar) [inline .env.template]: Can you add some documentation for these env vars?

**Review by harshkhandeparkar — COMMENTED** [review]
Any plans to add a rotate backup script?

**Review by harshkhandeparkar — COMMENTED** [review]
Have you tested this?

**Comment by shikharish** [comment]
Yes

**Review by harshkhandeparkar — APPROVED** [review]
Then I'll approve it but waiting for @rajivharlalka's approval before merging.

**Comment by proffapt** [comment]
@rajivharlalka look at this once, thanks.

File: backup/backup_to_dropbox.py
Comment (rajivharlalka) [inline backup/backup_to_dropbox.py:47]: assuming this works correctly.

File: backup/run_backup.sh
Comment (rajivharlalka) [inline backup/run_backup.sh]: I think this works for us right now, but shouldn't rotation should come before uploading the backup ? whatif the new backup has very high size and fails, i guess we should take this in consideration. Though shouldn't be an issue now.

File: docker-compose.yml
Comment (rajivharlalka) [inline docker-compose.yml]: tag version of postgres being used.

**Review by rajivharlalka — APPROVED** [review]

File: docker-compose.yml
Comment (shikharish) [reply docker-compose.yml]: we are using the latest version. shouldnt be a problem in the future i think.

File: backup/run_backup.sh
Comment (shikharish) [reply backup/run_backup.sh]: makes sense

File: backup/backup_to_dropbox.py
Comment (shikharish) [reply backup/backup_to_dropbox.py:47]: it does :)

**Review by harshkhandeparkar — APPROVED** [review]
I had approved this. If @rajivharlalka approves, it can be merged.

**Review by rajivharlalka — APPROVED** [review]

**Pull request merged**
